### DISCUSSION TITLE
add gradient colors

### DISCUSSION THREAD

**Discussion by lsaavedr** [makeswift/makeswift#778] [Ideas]
Hi, I want to share a preview of color selections in brandmark.io:

![Screenshot from 2024-06-08 21-15-46](https://github.com/makeswift/makeswift/assets/571849/9cf9d97e-315a-4d0e-8a8c-838833d45763)

In Makerswift we can save brand colors, it's very nice and simple, I love the kiss principle in all Makeswift stack, but it doesn't have gradient colors, it seems like a nice simple feature to add in Makerswift stack, a lot of brand colors need to be gradients colors,

Cheers!

**Comment by apledger** [comment]
Love this idea, thanks for sharing. How many saved gradients do you think you'll have in your design system?

**Reply by lsaavedr** [reply]
> Love this idea, thanks for sharing. How many saved gradients do you think you'll have in your design system?

1 or 2 gradients :P
